packages/tailwindcss/src/index.ts

```diff
@@ -397,7 +414,7 @@ export async function compile(
 }
 
 export async function __unstable__loadDesignSystem(css: string, opts: CompileOptions = {}) {
-  let result = await parseCss(css, opts)
+  let result = await parseCss(css, '/* @TODO */', opts)
```

Comment:
see my comment above about moving it to opts. Would also mean less API breakage before we're ready to actually rework the APIs.